Kajabi/pine
# DSS-470: chore:(link) - Housekeeping

# Description

Minor housekeeping-related adjustments to the link component including:

- Adding copy/descriptions for stories.
- Convert classes to function in order to prevent partial class names.
- Convert external icon to use `pds-icon`

Fixes [DSS-470](https://kajabi.atlassian.net/browse/DSS-470)

## Type of change

Please delete options that are not relevant.
If your type of change is not present, add that option.


# How Has This Been Tested?

1. Navigate to the link component and check impartial class names no longer appear in each story.
2. Check that the added docs descriptions make sense and are correct.



**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

# Checklist:

If not applicable, leave options unchecked.



[DSS-470]: https://kajabi.atlassian.net/browse/DSS-470?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
